code/datums/brain_damage/severe.dm

```diff
@@ -264,3 +264,36 @@
 	..()
 	if(prob(1) && !owner.has_status_effect(/datum/status_effect/trance))
 		owner.apply_status_effect(/datum/status_effect/trance, rand(100,300), FALSE)
+
+/datum/brain_trauma/severe/hypnotic_trigger
+	name = "Hypnotic Trigger"
+	desc = "Patient has a trigger phrase set in their subconscious that will trigger a suggestible trance-like state."
+	scan_desc = "latent cerebral loop"
```

Comment:
make this description be the same as the one for hypnotic stupor so that you can't just use a health analyzer to tell if the machine worked or not

Reply:
Good call